regress/README

```diff
@@ -11,50 +11,26 @@ with your production Catalog because the tables will all
 be deleted (dropped and recreated).
```

Comment:
We should refer to the systemtests in ctest and say that the regression tests are at deprecated.

Reply:
Added warning.